src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -44,16 +44,17 @@ def persist_flow_run(self, run_info: FlowRunInfo):
 
 
 class HealthyEnsuredProcess:
-    def __init__(self, executor_creation_func):
+    def __init__(self, executor_creation_func, context=None):
```

Comment:
It should not have a default None value here

Reply:
I see, fixed